### PR TITLE
Quadkey options

### PR DESCRIPTION
**Pull request by vincentsarago**
replace #41

### CODE REVIEW

File: cogeo_mosaic/utils.py [hunk 1]
```diff
@@ -237,7 +240,7 @@ def create_mosaic(
 
         maxzoom = max(maxzoom)
 
-    quadkey_zoom = minzoom
+    quadkey_zoom = quadkey_zoom or minzoom
```
Comment (vincentsarago) [inline cogeo_mosaic/utils.py:243]: @kylebarron I think this is the only change between your PR and mine

File: cogeo_mosaic/utils.py [hunk 2]
```diff
@@ -177,6 +177,7 @@ def create_mosaic(
     dataset_list: Sequence[str],
     minzoom: int = None,
     maxzoom: int = None,
+    quadkey_zoom: int = None,
```
Comment (vincentsarago) [inline cogeo_mosaic/utils.py:180]: Note there is a potential risque when a user will pass `quadkey_zoom` and `version==0.0.1`. But that's only on the API side (not on the CLI) so it's unlikely.

File: cogeo_mosaic/utils.py (hunk 1 above)
Comment (kylebarron) [reply cogeo_mosaic/utils.py:243]: I think
```py
quadkey_zoom = quadkey_zoom or minzoom
```
is simpler than
```py
    if quadkey_zoom is None:
        quadkey_zoom = minzoom
```

Comment (vincentsarago) [reply cogeo_mosaic/utils.py:243]: I know that's why I write it!

File: cogeo_mosaic/utils.py (hunk 2 above)
Comment (vincentsarago) [reply cogeo_mosaic/utils.py:180]: will be taken care of in #57

File: setup.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "requests",
     "rio-cogeo>=1.1.0",
     "rio-tiler>=2.0a4",
-    "rio-tiler-mosaic>=0.0.1dev3",
+    "rio-tiler-mosaic>=0.0.1dev4",
```
Comment (vincentsarago) [inline setup.py:15]: FYI, `0.0.1dev3` forced rio-tiler~1.2

**Pull request merged**